src/components/SubaccountNavigationButton/SubaccountNavigationButton.tsx

```diff
@@ -0,0 +1,126 @@
+
+    const jumpToSubaccount = useCallback(() => {
+      closeConfirmationBox();
+      setModalOpen(true);
+    }, [closeConfirmationBox, setModalOpen]);
+
+    const [offerButtonHidden, setOfferButtonHidden] = useLocalStorageSerializeKey(
+      ONE_CLICK_TRADING_OFFER_HIDDEN,
+      false
+    );
+    const [nativeTokenWarningHidden, setNativeTokenWarningHidden] = useLocalStorageSerializeKey(
+      ONE_CLICK_TRADING_NATIVE_TOKEN_WARN_HIDDEN,
+      false
+    );
+
+    const handleCloseOfferClick = useCallback(() => {
+      setOfferButtonHidden(true);
+    }, [setOfferButtonHidden]);
+
+    const handleCloseNativeTokenWarningClick = useCallback(() => {
+      setNativeTokenWarningHidden(true);
+    }, [setNativeTokenWarningHidden]);
+
+    const { remaining } = useSubaccountActionCounts();
+
+    const shouldShowInsufficientFundsButton = isSubaccountActive && insufficientFunds && !isNativeToken;
+    const shouldShowOfferButton = !isSubaccountActive && !offerButtonHidden && !isNativeToken;
+    const shouldShowAllowedActionsWarning = isSubaccountActive && remaining?.eq(0) && !isNativeToken;
+    const shouldShowNativeTokenWarning =
+      !tradeFlags?.isTrigger && isSubaccountActive && !nativeTokenWarningHidden && isNativeToken;
+
+    let content: ReactNode = null;
+    let onCloseClick: null | (() => void) = null;
+
+    const renderTooltipContent = useCallback(() => {
+      return (
+        <div onClick={(e) => e.stopPropagation()}>
+          <Trans>
+            Reduce wallet signing popups with One-Click Trading. This option is also available through the Wallet menu
+            in the top right. <ExternalLink href={SUBACCOUNT_DOCS_URL}>More Info</ExternalLink>.
+          </Trans>
+        </div>
+      );
+    }, []);
+
+    let clickable = true;
+
+    if (shouldShowNativeTokenWarning) {
+      const wrappedToken = getWrappedToken(chainId);
+      const nativeToken = getNativeToken(chainId);
+      clickable = false;
+      onCloseClick = handleCloseNativeTokenWarningClick;
+      content = (
+        <Trans>
+          One-Click Trading is not available using network's native token {nativeToken.symbol}. Consider using{" "}
+          {wrappedToken.symbol} as Pay token instead.
+        </Trans>
+      );
+    } else if (shouldShowAllowedActionsWarning) {
+      content = (
+        <Trans>
+          The previously authorized maximum number of Actions has been reached for One-Click Trading. Click here to
+          re-authorize.
+        </Trans>
+      );
+    } else if (shouldShowInsufficientFundsButton) {
+      content = (
+        <Trans>There are insufficient funds in your Subaccount for One-Click Trading. Click here to top-up.</Trans>
+      );
+    } else if (shouldShowOfferButton) {
+      onCloseClick = handleCloseOfferClick;
+      content = (
+        <TooltipWithPortal handle={<Trans>Enable One-Click Trading</Trans>} renderContent={renderTooltipContent} />
```

Comment:
On mobile when we will click on the tooltip it will not show the tootlip text but it will open the One Click Modal. We have fixed this issue on the Tooltip component using `onHandleClick`. So either you can use Tooltip component or add the solution to TooltipWithPortal

Reply:
fixed